### DISCUSSION TITLE
Anyone had issues with introspection not updating newly created tables in schema?

### DISCUSSION THREAD

**Discussion by brendan1212** [prisma/prisma#2570] [Q&A] [answered]
I'm having this issue. I don't assume there is a limit to the number of tables you can have in one schema, is there?

I can modify a table that already exists in my schema, but when I create a new table and introspect, it doesn't reflect in the schema at all. No errors.

Has anyone encountered this or know what could be the cause?

Thanks

(I also created an issue [here](https://github.com/prisma/prisma/issues/2569).)

**Comment by ryands17** [comment]
Hey @pachiraDIG :wave: 
I'm trying to reproduce this. In the meanwhile, could you update both `@prisma/cli` and `@prisma/client` to beta.6 and try again?

**Reply by brendan1212** [reply]
Hi @ryands17 . Thanks for the help. I can confirm the same issue with beta.6.

**Comment by brendan1212** [comment, marked as answer]
Found that this is a db user issue after all. Apologies and thanks for checking into this.

**Comment by ludralph** [comment]
Hi @brendan1212,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
